themes/cpmsoc/layouts/partials/footer.html

```diff
@@ -5,9 +5,29 @@
         &copy; UNSW Competitive Programming and Mathematics Society {{ now.Format "2006" }}
       </div>
       <div class="column is-4 is-12-mobile has-text-right">
-        <a href="https://www.facebook.com/CPMSoc/"><img class="small-social" src="/assets/icons/facebook_purple.png"></a>
-        <a href="https://discord.gg/9uxgxHY3pP"><img class="small-social" src="/assets/icons/discord_purple.png"></a>
-        <a href="mailto:cpmsoc@unswcpmsoc.com"><img class="small-social" src="/assets/icons/mail_purple.png"></a>
+        <a href="https://www.facebook.com/CPMSoc/">
+          <img class="small-social" src="/assets/icons/facebook_purple.png">
+        </a>
+        <a href="https://discord.gg/9uxgxHY3pP">
+          <img class="small-social" src="/assets/icons/discord_purple.png">
+        </a>
+        <a href="mailto:cpmsoc@unswcpmsoc.com">
+          <img class="small-social" src="/assets/icons/mail_purple.png">
+        </a>
```

Comment:
i would update the other icons while you're at it for consistency

Reply:
I reckon the order should be Facebook, Discord, Calendar, Email (email should be in an obvious spot so external organisations like sponsors can contact us as easily as possible)